File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessObjectReferenceIndirect_WithinIfStament_POI.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.ServiceTwo;

public class AccessObjectReferenceIndirect_WithinIfStament_POI {
	
	private ServiceTwo st = new ServiceTwo();

	public AccessObjectReferenceIndirect_WithinIfStament_POI(){
		if (st.getServiceOne().poi != null) {
			String s = "wrong";
			System.out.println(s);}
	}
}

File: testresources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessObjectReferenceIndirect_WithinIfStament_POI.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessObjectReferenceIndirect_WithinIfStament_POI
	{
        private Domain.Indirect.Intermediate.ServiceTwo st = new Domain.Indirect.Intermediate.ServiceTwo();

		public AccessObjectReferenceIndirect_WithinIfStament_POI()
		{
			if (st.GetServiceOne().poi != null)
			{
				string s = "wrong";
			}
		}
	}
}
